app/controllers/reviews.rb

```diff
@@ -0,0 +1,25 @@
+# get a list of comments
+get '/reviews' do
+end
+# get a form to add a new reviews
+get '/reviews/new' do
+
+  erb :new_review
+end
+# add a new reviews
+post '/reviews' do
+  Review.create(message: params[:message], shoe_id: session[:last_shoe_id], author_id:session[:user_id])
```

Comment:
Storing data like `last_shoe_id` in the session makes your actions non deterministic which means both you and the user will have a harder time understanding what actions are going to do. It's better to pass this information in as a param. 

Its helpful to think about users who might be using your app from multiple tab in the same session. You don't want one tab to affect the state of the other.